### ISSUE TITLE
Cannot find module ... Please verify that the package.json has a valid "main" entry.

### ISSUE THREAD

**Issue by Aterieur** [ProjectOpenSea/stream-js#97]
```
Error: Cannot find module '...\node_modules\@opensea\stream-js\dist\index.cjs'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:297:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:528:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:784:27)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at init_seaport (...:208:41)
    at main (...:601:10)
    at startThreads (...:658:9)
    at Object.<anonymous> (...:662:1)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)
    at MessagePort.<anonymous> (internal/main/worker_thread.js:139:25)
Init OpenSea SDK
```
Any quick fix for this?

**Comment by Meemaw** [comment]
Should be fixed on 0.0.18, can you check?

**Comment by itsthedoom** [comment]
@Aterieur This should have been fixed, closing this issue. If you are still running into issues please let us know.
